controllers/homeRoutes.js

```diff
@@ -37,37 +37,49 @@ router.get('/signup', (req, res) => {
 router.get('/space/:id', async (req, res) => {
   try {
     const spaceData = await Space.findByPk(req.params.id, {
-      include: Idea
+      include: Idea,
     });
     const space = spaceData.toJSON();
 
     res.render('space', { space });
-
   } catch (err) {
     res.status(400).json(err);
     console.log(err);
   }
 });
 
 // Create idea page
-router.get('/space/:space_id/idea', withAuth);
+router.post('/space/:space_id/idea', async (req, res) => {
```

Comment:
This should be a get route, not a post. The JS for the <form> will make a POST request to an API route to do the actual idea creation.

The api route could be something like /api/spaces/:space_id/ideas, as we will need the Space ID when creating an idea. Or we can just pass the space Id as part of the body data if we want to keep a simple /api/ideas route for the POST request.

Reply:
Or really, this route in particular should live in an API routes file. The get request needed here will be for rendering a view.

Reply:
Absolutely agree. I have the post route in api/ideas still, as well.

The JS for the POST wasn't reaching it because the fetch automatically is adding space/ to the front of my call.  Making it space/api/idea which doesn't exist.  
http://localhost:3001/space/api/idea 404 (Not Found) 
JS below
`  if (space_id && name && pitch) {
    const response = await fetch('api/idea',`

Reply:
Moving the form to a separate handlebars and re-linking up from there :) Will push up when complete